pkg/executor/wasm/errors.go

```diff
@@ -0,0 +1,13 @@
+package wasm
+
+import (
+	"github.com/bacalhau-project/bacalhau/pkg/models"
+)
+
+const WasmExecutorComponent = "Executor/Wasm"
```

Comment:
I don't mind having a standard for naming nested components, but lets make sure it plays nicely with string representation of base errors. I keep thinking that `Docker:ImageNotFound`, `S3Publisher:BucketNotFound` and `S3InputSource:BucketNotFound` as potential candidates. Adding `/` as a separator between component type and component name is valid, but just think it through

Reply:
Removed at the moment. Just going with executor.